server/src/models/feedback.es6

```diff
@@ -0,0 +1,27 @@
+
+
+/**
+ * Module for feedback model
+ * @module models/feedback
+ */
+
+const Sequelize = require('sequelize');
+
+const util = require('../services/util.es6');
+
+module.exports = util.getSequelizeConnection().define(
+  'feedbackEntries',
+  {
+    message: { type: Sequelize.STRING, field: 'message', allowNull: false },
```

Comment:
this is enough data for MVP, but we might want to flesh this out at some point to capture things like user info, email, role, etc..